patch/kconfig-exclusions

```diff
@@ -0,0 +1,115 @@
+CONFIG_REISERFS_FS
+CONFIG_JFS_FS
+CONFIG_XFS_FS
+CONFIG_GFS2_FS
+CONFIG_OCFS2_FS
+CONFIG_NILFS2_FS
+CONFIG_F2FS_FS
+CONFIG_ADFS_FS
+CONFIG_AFFS_FS
+CONFIG_BEFS_FS
+CONFIG_BFS_FS
+CONFIG_EFS_FS
+CONFIG_QNX4FS_FS
+CONFIG_QNX6FS_FS
+CONFIG_MINIX_FS
+CONFIG_VXFS_FS
+CONFIG_OMFS_FS
+CONFIG_ROMFS_FS
+CONFIG_SYSV_FS
+CONFIG_UFS_FS
+CONFIG_EXOFS_FS
+CONFIG_CEPH_FS
+CONFIG_NCP_FS
+CONFIG_CODA_FS
+CONFIG_AFS_FS
+CONFIG_NET_9P
+CONFIG_DLM
+CONFIG_QFMT_V1
+CONFIG_MTD
```

Comment:
we do use CONFIG_MTD on some platforms. patch/0042-armhf-additional-configs.patch